### DISCUSSION TITLE
nuclei-templates not found: version string empty

### DISCUSSION THREAD

**Discussion by blinky0** [projectdiscovery/nuclei#2575] [Q&A] [answered]
installed nuclei 2.7.4 by go method, tried to run a scan but templates are not found, directory of nuclei-templates are not existing , 
ran ut , ud command but didnt fix the issue

WRN] Use with caution. You are responsible for your actions.
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[INF] nuclei-templates are not installed, installing...
[ERR] Could not update templates: Version string empty

**Comment by abusabiha** [comment]
Here is also same problem. Did you fix your problem?

**Comment by ehsandeep** [comment]
@blinky0 make sure to install the latest version of the project, i.e 2.7.7 and if you still face the issue, please share the output of `nuclei -hc`

**Comment by forgedhallpass** [comment, marked as answer]
@blinky0 @abusabiha 
Can you confirm that the values within the configuration file are correct?
```
cat ~/.config/nuclei/.templates-config.json | jq
```

You could also try to rename the `.templates-config.json` to `.templates-config.json.bak` and let `nuclei` re-download the templates. See if that helps.

![image](https://user-images.githubusercontent.com/13679401/190384647-c54fa46f-87f2-4b2b-9697-1ba024873660.png)

**Comment by abusabiha** [comment]
![20220926_164215](https://user-images.githubusercontent.com/86252592/192257359-32f2cd7f-5d90-4895-a132-29b5827b8cac.jpg)
![20220926_164223](https://user-images.githubusercontent.com/86252592/192257442-27705a65-4ad5-4166-a7b5-267b31b69fb9.jpg)
![20220926_164238](https://user-images.githubusercontent.com/86252592/192257507-a65a0aca-e1c4-400d-9238-3c2ea07cae06.jpg)

**Comment by DanielRuf** [comment]
Same here.

.templates-config.json looks like this:

```json
{
    "nuclei-templates-directory": "/home/druf/nuclei-templates",
    "custom-s3-templates-directory": "/home/druf/nuclei-templates/s3",
    "custom-github-templates-directory": "/home/druf/nuclei-templates/github",
    "nuclei-version": "2.8.6",
    "nuclei-latest-version": "",
    "nuclei-templates-latest-version": ""
}
```

The hc output is this:

```
> ./nuclei_2.8.6_linux_amd64/nuclei -hc

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.8.6

		projectdiscovery.io

Version: 2.8.6
Operative System: linux
Architecture: amd64
Go Version: go1.19.4
Compiler: gc
File "/home/druf/.config/nuclei/config.yaml" Read => Ok
File "/home/druf/.config/nuclei/config.yaml" Write => Ok
File "/home/druf/.config/nuclei/.nuclei-ignore" Read => Ko (open /home/druf/.config/nuclei/.nuclei-ignore: no such file or directory)
File "/home/druf/.config/nuclei/.nuclei-ignore" Write => Ko (open /home/druf/.config/nuclei/.nuclei-ignore: no such file or directory)
File "/home/druf/nuclei-templates/.checksum" Read => Ko (open /home/druf/nuclei-templates/.checksum: no such file or directory)
File "/home/druf/nuclei-templates/.checksum" Write => Ko (open /home/druf/nuclei-templates/.checksum: no such file or directory)
IPv4 connectivity to scanme.sh:80 => Ok
IPv6 connectivity to scanme.sh:80 => Ko (dial tcp6 [2400:6180:0:d0::91:1001]:80: connect: network is unreachable)
IPv4 UDP connectivity to scanme.sh:53 => Ok

```

**Reply by DanielRuf** [reply]
Seems to be fixed with latest nuclei release.

**Reply by Steiner-254** [reply]
> Seems to be fixed with latest nuclei release.

Hey, and what was the root cause for that issue?

**Reply by DanielRuf** [reply]
Either try the accepted answer or a previous release.

**Reply by Steiner-254** [reply]
This _"try to rename the .templates-config.json to .templates-config.json.bak and let nuclei re-download the templates. See if that helps."_ ... will give it a trial.
